server/app/views/applicant/AddressCorrectionBlockTemplate.html

```diff
@@ -74,7 +74,9 @@ <h3 th:text="#{content.addressEntered}"></h3>
         ></button>
       </div>
     </form>
-    <div th:replace="~{applicant/ModalFragment:: modalContainer}"></div>
+    <div
+      th:replace="~{applicant/ModalFragment:: modalContainer(false, false)}"
```

Comment:
**question:** Can we include comments for the parameter names or is that not allowed in thymeleaf? Like `modalContainer(/*showProgramIndexLogin=*/false, /*showReviewLogin=*/false)`

Reply:
done. Although thymeleaf comments are a little more annoying because you have to do <!--/* showProgramIndexLogin= */-->, but 🤷‍♂️